Mathlib/Topology/MetricSpace/HausdorffDimension.lean

```diff
@@ -523,3 +523,14 @@ in `F`. -/
 theorem ContDiff.dense_compl_range_of_finrank_lt_finrank [FiniteDimensional ℝ F] {f : E → F}
     (h : ContDiff ℝ 1 f) (hEF : finrank ℝ E < finrank ℝ F) : Dense (range f)ᶜ :=
   dense_compl_of_dimH_lt_finrank <| h.dimH_range_le.trans_lt <| Nat.cast_lt.2 hEF
+
+/--
+The Hausdorff dimension of the orthogonal projection of a set `s` onto a subspace `K`
+is less than or equal to the Hausdorff dimension of `s`.
+-/
+theorem dimH_orthogonalProjection_le {𝕜 : Type u_1} {E : Type u_2} [RCLike 𝕜]
```

Comment:
```suggestion
theorem dimH_orthogonalProjection_le {𝕜 : Type u_1} [RCLike 𝕜]
```
I haven't tested this, but since E is used earlier, I guess you don't need it here either.